rpm/template.spec.em

```diff
@@ -1,3 +1,8 @@
+@{
+# Drop membership in rmw_implementation_packages group
+Provides = [p for p in Provides if p != 'ros-eloquent-rmw-implementation-packages(member)']
```

Comment:
Out of curisioty: where these values (ros-eloquent-rmw-implementation-packages(member),ros-eloquent-rmw-implementation-packages(all)) come from?

Reply:
Bloom is adding them to packages when the package.xml says that the package is a member of a group: https://github.com/ros-infrastructure/bloom/blob/d2265fd73f7c93437dc9fb704e893f6ef478b8ae/bloom/generators/rosrpm.py#L99-L101

Reply:
Long term, we can drop this patch and just not create this RPM at all, but as we discussed in the meeting, there are technical hurdles to making that possible.

Reply:
thanks for the info!